src/dfcx_scrapi/core/environments.py

```diff
@@ -293,44 +293,35 @@ def create_environment_by_display_name(
     def update_environment(
         self,
         environment_id: str,
-        environment_obj: types.Environment = None,
+        obj: types.Environment = None,
```

Comment:
@MRyderOC can we revert this change and leave `environment_obj` the same?  

Changing arg names will break backwards compat for anyone already running this code.  
Even if the arg name is non-standard, we don't want to break any existing user workflows.